choreolib/src/test/java/choreo/auto/AutoChooserTest.java

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Choreo contributors
+
+package choreo.auto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.wpi.first.hal.AllianceStationID;
+import edu.wpi.first.hal.HAL;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.simulation.DriverStationSim;
+import edu.wpi.first.wpilibj2.command.Commands;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class AutoChooserTest {
+  private static final String NONE_NAME = AutoChooser.NONE_NAME;
+
+  private String chooserPath(String testFuncName) {
+    return "/Test/AutoChooser/" + testFuncName;
+  }
+
+  private NetworkTable table(String testFuncName) {
+    // i'm unsure if this is needed but it won't hurt
+    NetworkTableInstance.getDefault().flushLocal();
```

Comment:
We shouldn't use the Default instance in tests we should create one in setup and destroy it after the test is completed